CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 ### Added
 
 - Spawn egg items for main Underling types, Consorts, and Carapacians
+- All aspect tree-related blocks moved to `AspectTreeBlocks`
```

Comment:
I'd leave this out since it's an internal code change and does not matter in any way for end users (AKA players).

Reply:
✅